### PR TITLE
feat: Added prometheus scrape_config_files support

### LINKED ISSUE #145: [prometheus] - New scrape_config_files
Starting from version 2.43.0 of Prometheus, we have a new section available in the configuration file prometheus.yml called 'scrape_config_files,' where you can specify the paths of files containing the scrape_config settings. 

Would it be possible to implement this functionality by adding the 'scrape_config_files' attribute to the prometheus.yml.j2 template if it is defined in a variable, for example, 'prometheus_scrape_config_files'? It could be a list of paths."

### PR DESCRIPTION
**Pull request by manzsolutions-lpr**
Add support for scrape_config_files (fixes #145)

Attention: This is merely a rebase of #292.

### CODE REVIEW

**Review by gardar — CHANGES_REQUESTED** [review]

**Review by gardar — CHANGES_REQUESTED** [review]
We need to make sure the `scrape_config_files` option is not configured for older versions that don't support the option.

File: roles/prometheus/meta/argument_specs.yml [hunk 1]
```diff
@@ -112,6 +112,7 @@ argument_specs:
       prometheus_scrape_configs:
         description:
           - "Prometheus scrape jobs provided in same format as in the L(official docs,https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config)"
+          - "This feature is available starting from Prometheus v2.43.0."
```
Comment (gardar) [inline roles/prometheus/meta/argument_specs.yml:115]:
```suggestion
```

File: roles/prometheus/meta/argument_specs.yml [hunk 2]
```diff
@@ -150,6 +151,14 @@ argument_specs:
         default:
           - "prometheus/targets/*.yml"
           - "prometheus/targets/*.json"
+      prometheus_scrape_config_files:
+        description:
+          - "List of folders where ansible will look for files containing custom scrape config configuration files which will be copied to C({{ prometheus_config_dir }}/scrapes/)."
```
Comment (gardar) [inline roles/prometheus/meta/argument_specs.yml:156]:
```suggestion
          - "List of folders where ansible will look for files containing custom scrape config configuration files which will be copied to C({{ prometheus_config_dir }}/scrapes/)."
           - "This feature is available starting from Prometheus v2.43.0."
```

**Review by gardar — CHANGES_REQUESTED** [review]
Looks like this line got placed under the wrong option

File: roles/prometheus/meta/argument_specs.yml (hunk 1 above)
Comment (manzsolutions-lpr) [reply roles/prometheus/meta/argument_specs.yml]: Snap, that one’s particularly embarrassing. Sorry and thank you for pointing it out!

**Review by gardar — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**